from database import DataBase
from exceptions import EntryNotFoundError


